src/core/src/core_logic/ConfigurePatching.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2020 Microsoft Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Requires Python 2.7+
+
+""" Configure Patching """
+from core.src.bootstrap.Constants import Constants
+
+
+class ConfigurePatching(object):
+    def __init__(self, env_layer, execution_config, composite_logger, telemetry_writer, status_handler, package_manager):
+        self.env_layer = env_layer
+        self.execution_config = execution_config
+
+        self.composite_logger = composite_logger
+        self.telemetry_writer = telemetry_writer
+        self.status_handler = status_handler
+
+        self.package_manager = package_manager
+
+    def start_configure_patching(self):
+        """ Start configure patching """
+        self.status_handler.set_current_operation(Constants.CONFIGURE_PATCHING)
+        self.raise_if_agent_incompatible()
```

Comment:
This needs to be considered further as a configurepatching request is rarely likely to get the attention of a customer. Bring it up for discussion at scrum.

Reply:
ToDo

Reply:
Resolved. 

Windows does not check for telemetry support. Created a ADO task for it. 
With the approach mentioned in the comment above, we log this error in Configure Patching substatus and continue with the uber-operation. So essentially, this check will fail again in assessment, where the operation will stop